apollo-client/src/test/java/com/ctrip/framework/apollo/spring/JavaConfigAnnotationTest.java

```diff
@@ -88,7 +88,8 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
 
     TestApolloConfigChangeListenerBean1 bean = getBean(TestApolloConfigChangeListenerBean1.class, AppConfig3.class);
 
-    assertEquals(3, applicationListeners.size());
```

Comment:
This change impacts a lot, so we should have more tests to cover the value change scenarios.

Reply:
You are righit.